File: 프로그래머스/Level1/최소직사각형.py
# https://programmers.co.kr/learn/courses/30/lessons/86491

def solution(sizes):
    for arr in sizes:
        arr.sort()
    a_max = -1
    b_max = -1
    for a, b in sizes:
        a_max = max(a, a_max)
        b_max = max(b, b_max)
    return a_max * b_max

File: 프로그래머스/Level1/최소직사각형.js
// https://programmers.co.kr/learn/courses/30/lessons/86491

const solution = (sizes) => {
	let minValue = -1;
	let maxValue = -1;
	sizes.forEach((arr) => {
		arr.sort((a, b) => a - b);
		maxValue = Math.max(maxValue, arr[1]);
		minValue = Math.max(minValue, arr[0]);
	});
	return minValue * maxValue;
};
